astropy/visualization/wcsaxes/tests/test_images.py

```diff
@@ -486,6 +486,18 @@ def test_ticks_labels(self):
 
         return fig
 
+    @figure_test
+    def test_no_ticks(self):
+        # Check that setting no ticks works
+        fig = plt.figure(figsize=(6, 6))
+        ax = fig.add_axes(
+            [0.1, 0.1, 0.8, 0.8], projection=WCS(self.msx_header), aspect="equal"
+        )
+        ax.set_xlim(-0.5, 148.5)
+        ax.set_ylim(-0.5, 148.5)
+        ax.coords[0].set_ticks(number=0)
+        return fig
```

Comment:
```suggestion
        # Draw to trigger calling `_update_ticks`.
        plt.draw()
        return fig
```
Without enforcing a `draw_wcsaxes` call I don't get the test to raise without your fixes installed.

Reply:
Hm, shouldn't `draw_wcsaxes()` be called somwhwere when the figure is rendered for the figure test? I think it's best to leave the `plt.draw()` call out for simplicity here. If any ticks are drawn, they'll show up in the figure test image.

Reply:
That's right, when creating the comparison figures, the actual methods are called. But that's only tested if `mpl_image_compare` is installed.
I've copied the `draw` from other tests below, which will ensure to always fail if rendering is broken, but I don't know if there are rules when to test a method in the regular image tests.